console/build.gradle

```diff
@@ -87,11 +87,11 @@ def gitRevisionDate() {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
 
-    compile 'me.leolin:ShortcutBadger:1.1.0+@aar'
+    compile 'me.leolin:ShortcutBadger:1.1.0@aar'
 
-    compile 'com.android.support:appcompat-v7:22.1.1'
-    compile 'com.android.support:cardview-v7:21.0.+'
-    compile 'com.android.support:recyclerview-v7:21.0.+'
+    compile 'com.android.support:appcompat-v7:23.1.1'
+    compile 'com.android.support:cardview-v7:23.1.1'
+    compile 'com.android.support:recyclerview-v7:23.1.1'
     compile 'com.google.android.gms:play-services-gcm:8.3.0'
```

Comment:
These google dependencies are out-of-date, but to upgrade them, I need to upgrade the `google-services` plugin. The next version of the `google-services` plugin is a beta, so it's probably safer to wait for its full release before these dependencies are upgraded